source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/ResourceInformationTests.cpp

```diff
@@ -253,7 +255,10 @@ namespace AdaptiveCardsSharedModelUnitTest
             auto actionRegistration = std::make_shared<ActionParserRegistration>();
             actionRegistration->AddParser("CustomActionWithImage", std::make_shared<TestCustomActionParser>());
 
-            ParseContext context(elementRegistration, actionRegistration);
+            auto featureRegistration = std::make_shared<FeatureRegistration>();
+            featureRegistration->AddFeature("CoolFeature", "1");
+
+            ParseContext context(elementRegistration, actionRegistration, featureRegistration);
```

Comment:
Any particular reason this is added to this test case?

Reply:
Just figured that since we were already dealing with registrations I'd drop it in here :)

---
In reply to: [272647844](https://github.com/Microsoft/AdaptiveCards/pull/2623#discussion_r272647844) [](ancestors = 272647844)

Reply:
n/m -- this was a mistake. fixing...

---
In reply to: [272664376](https://github.com/Microsoft/AdaptiveCards/pull/2623#discussion_r272664376) [](ancestors = 272664376,272647844)